src/Board.hs

```diff
@@ -101,9 +103,8 @@ isStartingLayout board = board == calculateStartingBoard board
 -- Return what a given board would look like in its unscrambled, initial
 -- layout.
 calculateStartingBoard :: Board -> Board
-calculateStartingBoard (Board d _) =
-    Board d $ generateOrderedLayout d
-    where generateOrderedLayout (Dimensions width height) =
-              let lastNumber = width * height - 1
-               in (map Just [1 .. lastNumber]) ++ [Nothing]
-
+calculateStartingBoard (Board d _) = Board d (generateOrderedLayout d)
+  where
+    generateOrderedLayout (Dimensions width height) =
+      let lastNumber = width * height - 1
+      in  (map Just [1 .. lastNumber]) ++ [Nothing]
```

Comment:
I'm not sure if this is universal, but [Chris Done's Haskell style guide](https://github.com/chrisdone/haskell-style-guide) does suggest two spaces.